### ISSUE TITLE
HomeKit secure Video on Apple TV - Unifi Protect

### ISSUE THREAD

**Issue by Cav19D** [hjdhjd/homebridge-unifi-protect#1099]
### Homebridge UniFi Protect Version

7.0.7

### Homebridge Platform and OS

Raspberry Pi 3B

### Homebridge Version

v1.8.4

### Node Version

v20.9.0

### UniFi OS Version

4.0.6

### UniFi Protect Controller Version

4.0.33

### Describe the problem

Starting with plugin ver 7.1.0 video on Apple TV is not playing at normal speed, it[laying back in slow motion instead. I checked my HomeKit setup and checked HomeBridge. I downgraded the Protect ver from ver 7.1.2 to version 7.0.5. Checked video playback and was back to normal speed on the Apple TV. I upgraded Proect plugin ver to 7.1.0 to confirm if issue started at version 7.1.0 which it.
Versions 7.1.0, 7.1.1 and 7.1.2 video stream is in slow motion. In dome instances the pop-up on the Apple TV popped ups delayed by 10 to 15 minutes after the event occurred.
Not sure if I provided correct info below., first time reporting an issue.
if you need more information please let me know.
Thank you, and thanks for a really good plug-in.

### Homebridge HBUP JSON configuration

```json
 "controllers": [
                {
                    "address": "192.00.00.0”,
                    "password": “1010101010101010”,
                    "username": “U”
                }
            ],
            "name": "UniFi Protect",
            "options": [
                "Disable.Video.Transcode.HighLatency.F09FC213891C",
                "Enable.Device.StatusLed.F09FC213891C",
                "Enable.Motion.SmartDetect.F492BF8CDBA9",
                "Enable.Motion.Trigger.F492BF8CDBA9",
                "Enable.Motion.SmartDetect.F4E2C60E771C"
            ],
            "_bridge": {
                "username": "0E:C3:6E:6C:D9:3B",
                "port": 59417
            },
            "platform": "UniFi Protect"
        },
```

### Relevant log output

```shell
homebridge-unifi-protect 7.1.2 /var/lib/homebridge/node_modules
[03/08/2024, 07:17:48] [UniFi Protect] Restarting child bridge...
[03/08/2024, 07:17:48] Got SIGTERM, shutting down child bridge process...
[03/08/2024, 07:17:56] [UniFi Protect] Child bridge process ended
[03/08/2024, 07:17:56] [UniFi Protect] Process Ended. Code: 143, Signal: null
[03/08/2024, 07:18:03] [UniFi Protect] Restarting Process...
[03/08/2024, 07:18:05] [UniFi Protect] Launched child bridge with PID 28515
[03/08/2024, 07:18:06] Registering platform 'homebridge-unifi-protect.UniFi Protect'
[03/08/2024, 07:18:06] [UniFi Protect] Loaded homebridge-unifi-protect v7.1.2 child bridge successfully
[03/08/2024, 07:18:06] Loaded 6 cached accessories from cachedAccessories.0EC36E6CD93B.
[03/08/2024, 07:18:07] [UniFi Protect] Using FFmpeg version: 6.0-homebridge-alpine-armv7l-static.
[03/08/2024, 07:18:07] Homebridge v1.8.4 (HAP v0.12.2) (UniFi Protect) is running on port 59417.
```

### Acknowledgment that you are only running UniFi OS and UniFi Protect releases from the Ubiquiti Official release channel

- [X] I acknowledge that I am not running any early access / beta firmwares or operating systems from either Ubiquiti or Apple.

**Comment by hjdhjd** [comment]
Thanks for the report. I’d strongly encourage you to read the changelog / release notes closely for each release. The [changelog for v7.1.0](https://github.com/hjdhjd/homebridge-unifi-protect/blob/main/docs/Changelog.md#710-2024-06-15), states (it’s the first bullet point in fact) that transcoding is now enabled by default whether for low or high latency scenarios. If you want to change the default, disable transcoding, which you’ve done but only for high latency situations.

All that said: Pi 3 (and Pi in general) is not a great platform for having anything approaching an optimal experience with HBUP and I’d advise you to upgrade to something beefier, preferably an Apple Silicon-era Mac.

TL;DR: you need to disable transcoding because the defaults changed in v7.1.0, as documented.

**Comment by Cav19D** [comment]
Thank you for your quick reply to my report. I usually read the release notes  but I must confess I didn’t on last couple of updates.
I will take your instructions and change the settings you mentioned. Going to look into your suggestion of running HB on something other that a Raspberry Pi and to read the release notes.
Also, I really appreciate the work you put into the Unifi Protect plugin.

Regards

Cav19D

**Comment by hjdhjd** [comment]
Hope you enjoy it. You’re encountering challenges because you’re using seriously underpowered hardware to do things that definitively require more processing heft than what the noble-but-quite-constrained Pi can do, especially at scale.

Thanks for the kind words…best of luck!
